soc/arm/nxp_lpc/lpc55xxx/Kconfig.series

```diff
@@ -14,5 +14,6 @@ config SOC_SERIES_LPC55XXX
 	select SOC_FAMILY_LPC
 	select CPU_CORTEX_M_HAS_SYSTICK
 	select CPU_CORTEX_M_HAS_DWT
+	select HAS_MCUX_SDIF
```

Comment:
At some point you may want to follow the approach here instead:

https://github.com/zephyrproject-rtos/zephyr/blob/8e4107f9be8a554c1c611b9b7ca665be8741f3b5/soc/arm/nordic_nrf/Kconfig.peripherals#L1

This scheme avoids duplicated work (since the devicetree already has the information you need, and you can initialize the Kconfig options based on what's in the devicetree) and creates a single point of truth.

Reply:
Thank you for making me aware of this. While this approach is probably not in the scope of this PR, I agree it is a cleaner way to handle the presence of IP blocks on the SOC. @dleach02 FYI- we may want to consider switching to this approach for setting Kconfig symbols to indicate the presence of a peripheral

Reply:
This is unused now that you have followed @galak's suggestions; shouldn't you drop this patch?